.gitignore

```
@@ -1,11 +1,14 @@
 # Ignore the build directory
-/build/
+build
 
 # Ignore the content of the debug folder
 debug/*json
 
+# Ignore the output of the setup (ie: Proving and Verifying key)
+/zecale_setup/
+
 # Ignore node_modules
-/zecale_contracts/node_modules/*
+contracts/node_modules/*
```

```suggestion
/contracts/node_modules/*
```
(to match at the repo root only)